src/SourceBuild/tarball/content/tools-local/init-build.proj

```diff
@@ -118,6 +120,14 @@
     <WriteLinesToFile File="$(CompletedSemaphorePath)BuildXPlatTasks.complete" Overwrite="true" />
   </Target>
 
+  <Target Name="PatchPackageVersions">
```

Comment:
Any objections to adding a TODO comment to clean this up and link to https://github.com/dotnet/installer/issues/14492?

Reply:
#done